src/App.js

```diff
@@ -1,12 +1,164 @@
 import React from 'react';
 import './App.css';
+import {useForm} from "react-hook-form";
+import Button from "./components/Button";
+import Fruitcounter from "./components/Fruitcounters";
+
+
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    const [banaanAmount, setBanaanAmount] = React.useState(0);
+    const [aardbeiAmount, setAardbeiAmount] = React.useState(0);
+    const [appelAmount, setAppelAmount] = React.useState(0);
+    const [kiwiAmount, setKiwiAmount] = React.useState(0);
+
+    const { register, handleSubmit } = useForm();
+
+
+    const logResults = (data) => {
+        console.log(data)
+        console.log(`Er zijn ${banaanAmount}bananen besteld`, aardbeiAmount, appelAmount, kiwiAmount)
+    }
+
+    const resetFruitAmount = () => {
+        setBanaanAmount(0)
+        setAardbeiAmount(0)
+        setAppelAmount(0)
+        setKiwiAmount(0)
+    }
+
+    return (
+        <>
+            <main>
+                <section id="outer-container">
+                    <article id="inner-container">
+                            <div className="fruit">
+                                <h1>Fruitmand bezorgservice</h1>
+
+                                <h2>🍌 Bananen</h2>
+
+                                <Fruitcounter
+                                    fruit={banaanAmount}
+                                    setFruit={setBanaanAmount}
+                                    fruitId='banaanCounter'
+                                ></Fruitcounter>
```

Comment:
Je zou misschien de tekst mee kunnen geven aan het FruitCounter component en dan daar het h2 element terug laten geven.
```suggestion
                                <Fruitcounter
                                    fruit={banaanAmount}
                                    setFruit={setBanaanAmount}
                                    fruitId='banaanCounter'
                                    name="🍌 Bananen"
                                ></Fruitcounter>
```